### PR TITLE
Add new breed classifications and breeds for Cellosaurus request

### LINKED ISSUE #187: Add new NCBI Taxon for Cellosaurus request
I would like to add the following new NCBITaxon to VBO:

NCBITaxon:59729, Taeniopygia guttata (Zebra finch)
NCBITaxon:10036, Mesocricetus auratus (Golden hamster)
NCBITaxon:186623, Actinopteri (ray-finned fish)
NCBITaxon:8022, Oncorhynchus mykiss (Rainbow trout)
NCBITaxon:7955, Danio rerio (Zebrafish)
NCBITaxon:7962, Cyprinus carpio (Common carp)
NCBITaxon:7957, Carassius auratus (Goldfish)
NCBITaxon:8090, Oryzias latipes (Japanese rice fish)
NCBITaxon:8353, Xenopus (Clawed frog)
NCBITaxon:8364, Xenopus tropicalis (Western clawed frog)

Confirmed these taxon are in https://obofoundry.org/ontology/ncbitaxon.html

### PR DESCRIPTION
**Pull request by katiermullen**
Closes #187

Addresses #181

### CODE REVIEW

File: src/ontology/components/highlevelclass.tsv [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
+VBO:0400047	Clawed frog breed	VBO:0400000	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153			Vertebrate breed of the taxon Xenopus. A vertebrate breed is a group of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) that distinguish it from other organisms of the same species, and/or for which cultural or geographical separation has led to the general acceptance of its separate identity. Breeds are formed through genetic isolation and either natural adaptation to the environment or selective breeding, or a combination of the two.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153		NCBITaxon:8353		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/
```
Comment (sabrinatoro) [inline src/ontology/components/highlevelclass.tsv:47]: why "clawed frog breed" and not more general "frog breed"?

File: src/ontology/components/highlevelclass.tsv [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+VBO:0400041	Actinopteri breed	VBO:0400000	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153			Vertebrate breed of the taxon Actinopteri comprised of all extant ray-finned fish minus the bichirs. A vertebrate breed is a group of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) that distinguish it from other organisms of the same species, and/or for which cultural or geographical separation has led to the general acceptance of its separate identity. Breeds are formed through genetic isolation and either natural adaptation to the environment or selective breeding, or a combination of the two.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153	https://en.wikipedia.org/wiki/Actinopteri	NCBITaxon:186623		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/
```
Comment (sabrinatoro) [inline src/ontology/components/highlevelclass.tsv:41]: Could we call this term "fish breed"?

File: src/ontology/components/livestockbreeds.tsv [hunk 3]
```diff
@@ -607,4 +607,31 @@ VBO:0017480	Zemmour (Sheep)		Zemmour	Sheep		240514-128	https://github.com/monarc
...
+VBO:0017495	AB (Zebrafish)		AB	Zebrafish		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:7955		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				AB	http://purl.obolibrary.org/obo/vbo#most_common_name		ZDB-GENO-960809-7																																															ZDB-GENO-960809-7	https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648
```
Comment (sabrinatoro) [inline src/ontology/components/livestockbreeds.tsv:621]: the source should be ZFIN:ZDB-GENO-990308-9
(you forgot the "ZFIN:" prefix)

File: src/ontology/components/livestockbreeds.tsv [hunk 4]
```diff
@@ -607,4 +607,31 @@ VBO:0017480	Zemmour (Sheep)		Zemmour	Sheep		240514-128	https://github.com/monarc
...
+VBO:0017497	SJD (Zebrafish)		SJD	Zebrafish		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:7955		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				SJD	http://purl.obolibrary.org/obo/vbo#most_common_name		ZDB-GENO-990308-9																																															ZDB-GENO-990308-9	https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648
```
Comment (sabrinatoro) [inline src/ontology/components/livestockbreeds.tsv:623]: the source should be ZFIN:ZDB-GENO-990308-9
(you forgot the "ZFIN:" prefix)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
- should we put all of these new breeds in the "livestock" spreadsheet? Maybe we need a new one? 
- please see comments. It might be easier to review these changes together

File: src/ontology/components/highlevelclass.tsv (hunk 1 above)
Comment (katiermullen) [reply src/ontology/components/highlevelclass.tsv:47]: changed to frog breed

File: src/ontology/components/highlevelclass.tsv (hunk 2 above)
Comment (katiermullen) [reply src/ontology/components/highlevelclass.tsv:41]: yes, changed to NCBITaxon:32443 and term label to "fish breed"

File: src/ontology/components/livestockbreeds.tsv (hunk 4 above)
Comment (katiermullen) [reply src/ontology/components/livestockbreeds.tsv:623]: Added

File: src/ontology/components/livestockbreeds.tsv (hunk 3 above)
Comment (katiermullen) [reply src/ontology/components/livestockbreeds.tsv:621]: done

**Review by sabrinatoro — APPROVED** [review]
Thank you @katiermullen! It looks great! 
I am approving and merging the PR. 

Note that I saw that "common carp breed" does not have any children. I think it is ok because the term requested map to species and not breeds.

**Pull request merged**